src/test/data/JsonSerializableAddressBookTest/typicalParticipantsAddressBook.json

```diff
@@ -0,0 +1,107 @@
+{
+  "_comment": "AddressBook save file which contains the same Person values as in TypicalPersons#getTypicalAddressBook()",
```

Comment:
Should change 'Person' to 'Participant'

Reply:
Also note the filename. Shouldn't be camelCase for filenames :sweat_smile:

Reply:
all the test filenames are camelCase so I think they intentionally have different formats for these files. Java class file -> starts with a capital letter.